package.json

```diff
@@ -16,23 +16,34 @@
   },
   "dependencies": {
     "lodash": "3.10.1",
+    "perry": "0.1.3",
     "promise": "7.1.1",
-    "superagent": "1.6.1"
+    "superagent": "1.7.1"
   },
   "devDependencies": {
-    "blanket": "1.1.7",
-    "browserify": "12.0.1",
-    "coveralls": "2.11.2",
+    "blanket": "1.2.1",
+    "browserify": "13.0.0",
+    "coveralls": "2.11.6",
     "eslint": "0.24.0",
-    "jsonapi-server": "1.0.3",
+    "jsonapi-server": "1.2.0",
+    "karma": "0.13.19",
+    "karma-chrome-launcher": "0.2.2",
+    "karma-firefox-launcher": "0.1.7",
+    "karma-mocha": "0.2.1",
+    "karma-phantomjs-launcher": "0.2.3",
     "mocha": "2.2.5",
     "mocha-lcov-reporter": "0.0.2",
+    "phantomjs": "1.9.19",
     "plato": "1.5.0",
     "uglifyjs": "2.4.10"
   },
   "scripts": {
     "build": "./node_modules/browserify/bin/cmd.js lib/Client.js | ./node_modules/uglifyjs/bin/uglifyjs -cm > dist/jsonapi-client.min.js",
-    "test": "./node_modules/mocha/bin/mocha -R spec ./test/*.js",
+    "mocha": "./node_modules/mocha/bin/mocha -R spec ./test/*.js",
+    "prekarma": "node ./node_modules/jsonapi-server/example/server.js &",
+    "karma": "sleep 1 && ./node_modules/browserify/bin/cmd.js --ignore jsonapi-server/example/server.js test/*.js > dist/jsonapi-client-test.js && ./node_modules/karma/bin/karma start ./karma.conf.js",
```

Comment:
This has an `--ignore` line which prevents the Browserify build from trying to bundle jsonapi-server into the payload sent to the browser via Karma.